selfservice/hook/web_hook.go

```diff
@@ -87,6 +96,16 @@ type (
 	rawHookResponse struct {
 		Messages []errorMessage `json:"messages"`
 	}
+
+	// TODO fandom - fix webhooks
```

Comment:
Note: I patched it in https://github.com/Wikia/kratos/pull/97/commits/181811732626adda44cb0198e0e89f45b2ec5a7b
But I didn't test it